docs/reference/release-notes/6.8.asciidoc

```diff
@@ -3,13 +3,44 @@
 
 coming::[6.8.11]
```

Comment:
> coming::[6.8.11]

If you can remove this "coming" macro in this same PR, that would be great.  Otherwise, LGTM